provider/scripts/spark/offline_store_spark_runner.py

```diff
@@ -458,9 +458,9 @@ def execute_df_job(output_uri, code, store_type, spark_configs, credentials, sou
 def _validate_output_df(output_df):
     if output_df is None:
         raise Exception("the transformation code returned None.")
-    if not isinstance(output_df, DataFrame):
+    if type(output_df).__name__ != "DataFrame":
```

Comment:
is this backwards compatible with the other version?

Reply:
Yes, this will handle both imports paths. I've checked with both versions of Spark.